drivers/mipi_dbi/CMakeLists.txt

```diff
@@ -5,3 +5,15 @@
 zephyr_sources_ifdef(CONFIG_MIPI_DBI_SPI mipi_dbi_spi.c)
 zephyr_sources_ifdef(CONFIG_MIPI_DBI_SMARTBOND mipi_dbi_smartbond.c)
 zephyr_sources_ifdef(CONFIG_MIPI_DBI_NXP_LCDIC mipi_dbi_nxp_lcdic.c)
+zephyr_sources_ifdef(CONFIG_MIPI_DBI_NXP_FLEXIO_LCDIF mipi_dbi_nxp_flexio_lcdif.c)
+# Data bus width is used by the SDK driver and processes it as a compile time option
+if(CONFIG_MIPI_DBI_NXP_FLEXIO_LCDIF)
+  dt_chosen(flexio0_lcd PROPERTY "zephyr,display")
+  dt_prop(data_bus_width PATH "${flexio0_lcd}" PROPERTY "mipi-mode")
+  # Values for mipi-mode property are defined inside dt-bindings/mipi_dbi/mipi_dbi.h.
+  # We pass a define to the SDK driver if we are using 8-bit mode.
+  if((data_bus_width EQUAL 8) OR (data_bus_width EQUAL 5))
```

Comment:
Wouldn't `data_bus_width` being 8 indicate 16 bit mode?

Reply:
This is looking at the values in the mipi_dbi.h file.

Reply:
Yes, but in both cases (`data_bus_width==8` or `data_bus_width==5`) we are setting the FLEXIO bus width to 8. Is this correct, or should we set it to 16 when `data_bus_width==8`?

Reply:
Yes, this is accurate one for 8080 and one for 6800

Reply:
After some offline discussion, I understand this better now- the SDK driver will default to 16 bits if no value is set for this variable. The enum values `5` and `8` correspond to MIPI DBI 8 bit mode for 6800 and 8080. Since the SDK defaults to 16 bit mode, no `else` statement is needed here to set the definition in that case